docker-compose.full.yml

```diff
@@ -63,6 +63,7 @@ services:
       - "5000"
     volumes:
       - static_data:/opt/invenio/var/instance/static
+      - ../fare_shared_data/fare_files:/opt/invenio/instance/data/
```

Comment:
This changes are not necessary anymore. Please revert them and rebase to the new master. Thanks

Reply:
ok, i'll apply the changes in the fare.yml file and revert the changes here